.gitignore

```diff
@@ -0,0 +1 @@
+rolecall/**/*.png
```

Comment:
is this necessary?

Reply:
not anymore. Will delete this line from the ignore file

Reply:
fixed issue as of c670fcf